ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -3087,6 +3087,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-upi-zones
+- as: aws-ipi-p3-f28-long-duration-workload
```

Comment:
Would you mind to sort them in alphanumeric order ?

Reply:
@liangxia Done